src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="Apple A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" 
+        width="160px" 
+        height="134px"
+        class="card__img"
+      >
+      <h1 class="card__name">APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)</h1>
+      <div class="card__code">Product code: 195434</div>  
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__review">Reviews: 5</div>
+      </div>
+      <div class="card__price">
```

Comment:
```suggestion
      <div class="card__code">Product code: 195434</div>  
      
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        
        <div class="card__review">Reviews: 5</div>
      </div>
      
      <div class="card__price">
```
Add newlines between blocks of code to make it easier to read. Also, try to use semantic tags instead of divs where it's possible 
![image](https://user-images.githubusercontent.com/96409563/232689075-6d8109aa-4506-423e-8f89-d62b4737049f.png)